document/core/exec/instructions.rst

```diff
@@ -112,6 +112,12 @@ Where the underlying operators are non-deterministic, because they may return on
    \end{array}
 
 
+Integer, Floating-point Numeric instructions
+............................................
+
+The below subset of numeric instructions is applicable only to operands with :ref:`integer <syntax-int>` and :ref:`floating point <syntax-float>` value types, and not to operands of Fixed-width SIMD value type.
```

Comment:
This comment and heading seems unnecessary. Either it already follows from syntax, because there are no SIMD testops; or there are, in which case the rules would need explicit side conditions / case distinctions anyway that control what they apply to.